recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
Thanks for clarifying.

Actually, it is quite the other way around. `nbconvert` requires LaTeX to build PDFs. Though `nbconvert` does not have a hard requirement on LaTeX. It simply fails to build PDFs if LaTeX is not present.